src/cli/index.ts

```diff
@@ -19,7 +19,7 @@ import { registerTelemetry } from "./commands";
 
 export * from "./commands";
 
-const pkg = require("../../package.json");
+export const pkg = require("../../package.json");
```

Comment:
why do we need to export the package.json?

Reply:
As we are using this 'pkg' in all the commands to collect CLI-Version field, instead of importing in all commands' files, I exported it from index.ts so it can be used wherever wanted.